app/controllers/admin/shelters_controller.rb

```diff
@@ -1,7 +1,7 @@
 class Admin::SheltersController < ApplicationController
   def index
     @shelters = Shelter.order_alphabetically_reverse
-    @pending_shelters = Shelter.select_pending_shelters
+    @pending_shelters = Shelter.select_pending_shelters.order_alphabetically
```

Comment:
Could put this inside the select_pending_shelters method as well, but this works!